src/app/service/event.service.ts

```diff
@@ -3,58 +3,56 @@ import { HttpClient, HttpParams } from '@angular/common/http';
 import { BehaviorSubject, Observable, catchError, finalize } from 'rxjs';
 import { EventRequest } from '../entity/EventRequest';
 import { UserProfile } from '../entity/UserProfile';
+import { Bet } from '../entity/Bet';
 
 @Injectable({
   providedIn: 'root',
 })
 export class EventService {
-  private apiUrl = 'http://localhost:8080/api/v1/events'; // Adjust the base URL as needed
-  private addUrl = 'http://localhost:8080/api/v1/events/add';
-  private getEventsUrl = 'http://localhost:8080/api/v1/events'; // Adjust the URL as needed
+  private apiUrl = 'http://localhost:8080/api/v1/events';
 
   private loadingSubject = new BehaviorSubject<boolean>(false);
   public loading$: Observable<boolean> = this.loadingSubject.asObservable();
 
   constructor(private http: HttpClient) {}
 
   addEvent(eventRequest: EventRequest): Observable<any> {
-    return this.http.post(this.addUrl, eventRequest);
+    return this.http.post('http://localhost:8080/api/v1/events/add', eventRequest);
+  }
```

Comment:
Why doesn't this use the apiUrl variable ? Why is the url hardcoded ?

Reply:
it was a requested change